### PR TITLE
Add goals

### PR DESCRIPTION
**Pull request by CheezItMan**
# What's Changed
[] I've added testing learning goals to waves 2-3.

## Feedback requested
Let me know if there's something to change.

### CODE REVIEW

**Comment by kariabancroft** [comment]
Looks good!

**Comment by Hamled** [comment]
I might be misunderstanding, but with the new wave 3 learning goal that students add their own testing files should we no longer be giving them the stub files `checking_account_spec.rb` and `savings_account_spec.rb`?

**Comment by droberts-sea** [comment]
Yeah, they should be implementing tests but not coming up with their own yet.

File: README.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ This type of table, where records from other tables are associated with each oth
 ### Learning Goals
 - Use inheritance to share some behavior across classes
 - Enhance functionality built in Wave 1
-- Add your own testing files to test new classes and inherited functionality.
+- Add your own tests new classes and inherited functionality.
```
Comment (Hamled) [inline README.md]: "Add your own tests _for all_ ..."

**Pull request merged**